### DISCUSSION TITLE
init_surf only two steps?

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#661] [General]
Dear all,
I want to use init_ surf to get some data, but init_surf was found only has the stage 1 and the stage 2,。
So i wangt to konw init_surf only generates the corresponding POSCAR, and does not run MD，right?
![image](https://user-images.githubusercontent.com/53855907/157487669-fe3190c6-13cc-4287-9732-9189bb5cc2d6.png)

![image](https://user-images.githubusercontent.com/53855907/157489263-30393322-1b5a-4cd6-b5cf-1b23f898a038.png)

Looking forward to your reply.
Saber

**Comment by AnguseZhang** [comment]
sure

**Reply by Saberve** [reply]
OK.Then I try to change the code myself to run stage 3 and stage 4.
Thank you for your reply.

**Reply by AnguseZhang** [reply]
We used to run iterations for bulk systems  first, and add structures for surface. Under such circumstances, we don't need too much AIMD data from scratch. Instead we add surf data by explorations in dpgen. So that's why we don't provide the function. But you're welcome to contribute your implementations if you would.

Best,
Yuzhi

**Reply by Saberve** [reply]
Thank you very much for your answer.
